client/webserver/site/webpack/prod.js

```diff
@@ -46,7 +46,7 @@ module.exports = merge(common, {
         exclude: /node_modules/,
         loader: 'eslint-loader',
         options: {
-          formatter: require('eslint/lib/cli-engine/formatters/stylish')
+          formatter: 'stylish'
```

Comment:
`'stylish'` also appears to be the default.

Reply:
Hmm, so it is.  OK to just keep this in case a future breaking version changes the default and whoever gets to do that update doesn't realize?